perf/test/angular-cli/tslint.json

```diff
@@ -1,32 +1,35 @@
 {
```

Comment:
We should change tslint to eslint in the future 😩

Reply:
Are we actively linting? I just put the default Angular CLI settings here.

ESLint support is not coming to Angular CLI anytime soon:
https://github.com/angular/angular-cli/issues/13732

Reply:
@kmdrGroch please note that the Angular CLI has its own lifecycle. Its a bit like `react-scripts`, except react has only 1 dependency (`react-scripts`), which brings its own versions, Angular took another approach, where it updates versions when you do an `ng create` or `ng update`. 

>Are we actively linting? 

Nope, we aren't linting. It's only there in order to be as much as a real-world project as we can.

> ESLint support is not coming to Angular CLI anytime soon

That's a bummer 😯

Reply:
"Nope, we aren't linting. It's only there in order to be as much as a real-world project as we can" that's why we should change to eslint in the future. Tslint is no longer developed. They said to change to eslint :)

Reply:
Actually, the unexpected happened today when the Angular team decided to endorse a community-created plugin for using ESLint with Angular CLI rather than replacing the TSLint plugin that come with Angular CLI:
https://github.com/angular/angular-cli/pull/18677#issuecomment-707251785